app/src/main/java/org/mozilla/focus/utils/Settings.java

```diff
@@ -57,6 +55,12 @@ public boolean shouldSaveToRemovableStorage() {
         return defined[0].equals(value); // assume the first item is for removable storage
     }
 
+    public boolean shouldUseSpeedMode(){
+        return preferences.getBoolean(
+                resources.getString(R.string.pref_key_privacy_block_speed_mode),
+                true);
+    }
```

Comment:
The naming is inconsistent here, shouldUseSpeedMode is decided by privacy_**block**_speed_mode?